library/HTMLPurifier/URIFilter/Munge.php

```diff
@@ -100,11 +100,11 @@ protected function makeReplace($uri, $config, $context)
         $string = $uri->toString();
         // always available
         $this->replace['%s'] = $string;
-        $this->replace['%r'] = $context->get('EmbeddedURI', true);
-        $token = $context->get('CurrentToken', true);
-        $this->replace['%n'] = $token ? $token->name : null;
-        $this->replace['%m'] = $context->get('CurrentAttr', true);
-        $this->replace['%p'] = $context->get('CurrentCSSProperty', true);
+        $this->replace['%r'] = $context->get('EmbeddedURI', true) ?: '';
```

Comment:
huh, I never knew ?: worked even in PHP 5 LOL

Reply:
@ezyang It's the ternary operator or "elvis operator", since PHP 5.3 you can leave out the middle part :) - https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary